app/assets/javascripts/type_station/editables/text_html.js.coffee

```diff
@@ -3,6 +3,7 @@ class window.TS.EditableText
     @editor = new MediumEditor @elements,
       disableReturn: true
       buttons: ['bold', 'italic', 'underline', 'anchor']
+      buttonLabels: 'fontawesome'
```

Comment:
Should use Ionic Framework icons so that the user can use whatever version of fontawesome they like